docs/metrics/prometheus/list.md

```diff
@@ -290,6 +296,11 @@ For deployments with [Site Replication](https://min.io/docs/minio/linux/operatio
 | `minio_bucket_replication_sent_bytes`               | Total number of bytes replicated to the target bucket.                           |
 | `minio_bucket_replication_sent_count`               | Total number of objects replicated to the target bucket.                         |
 | `minio_bucket_replication_credential_errors`        | Total number of replication credential errors since server start                 |
+| `minio_bucket_replication_proxied_get_requests_count` | Number of GET requests proxied to replication target                          |
+| `minio_bucket_replication_proxied_head_requests_count` | Number of HEAD requests proxied to replication target                          |
+| `minio_bucket_replication_proxied_delete_tagging_requests_count` | Number of DELETE requests proxied to replication target                          |
+| `minio_bucket_replication_proxied_get_tagging_requests_count` | Number of GET tagging requests proxied to replication target                          |
+| `minio_bucket_replication_proxied_put_tagging_requests_count` | Number of PUT tagging requests proxied to replication target                          |
```

Comment:
```suggestion
| `minio_bucket_replication_proxied_put_tagging_requests_total` | Number of PUT tagging requests proxied to replication target                          |
```

Reply:
We should also capture the proxies that failed and if the failed is increasing or not.